public/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/index.css" />
+    <title>Form JS Challenge</title>
+  </head>
+  <body>
```

Comment:
dodaj sobie jakiś div "wrapper" tak aby można było względem niego pozycjonować form za pomocą:
```
margin: 0 auto;
max-width: ...
```
a nie absoluta jak to robisz teraz :)